packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -126,18 +159,52 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     });
   }
 
+  _onInputTriggerTypingEventsHandler(e: InputEvent): void {
+    if (this._isValuableTextEntered()) {
+      this._throttledTriggerTypingStartAction?.({ event: e.event });
```

Comment:
What are the reasons for us to limit the number of times the typingStart event is triggered?

Reply:
Moreover, it looks like now if we trigger first input event at time `0` and the next one - at time `1499` ms, second input will be ignored, so at time `2000` end event will be still triggered cause no timer update is happened, right? It looks incorrect to me - i think each input event should affect time "to end"
P.s. "each input event should affect time to end" !== "each input event recreates a timeout"

Reply:
We decided to drop the throttle functionality [30f954d8b2d174191c45c26c9ef35cce6e9da515]